### ISSUE TITLE
UI not working?

### ISSUE THREAD

**Issue by kwalcock** [lum-ai/odinson#137]
I wanted to see how the state interacted with the UI and followed instructions at https://github.com/lum-ai/odinson/wiki/Using-the-State using the mentioned commit and instructions in the README file of the ui directory.  Everything seemed to install OK even on Windows, but it didn't appear to run correctly.  Before I spend time to figure it out, I'm going to ask if this looks at all familiar.  It appears that Gus wrote the wiki page, so I'll start there.  Thanks.

```
PS E:\Users\kwa\Clients\lum\Projects\odinson-project\odinson\ui> npm run dev

> odinson-ui@0.1.0 dev E:\Users\kwa\Clients\lum\Projects\odinson-project\odinson\ui
> concurrently "npm run server" "npm run client"

[0]
[0] > odinson-ui@0.1.0 server E:\Users\kwa\Clients\lum\Projects\odinson-project\odinson\ui
[0] > nodemon src/server/index.js
[0]
[1]
[1] > odinson-ui@0.1.0 client E:\Users\kwa\Clients\lum\Projects\odinson-project\odinson\ui
[1] > webpack-dev-server --mode development --devtool inline-source-map --hot
[1]
[0] [nodemon] 1.19.4
[0] [nodemon] to restart at any time, enter `rs`
[0] [nodemon] watching dir(s): src\server\**\*
[0] [nodemon] watching extensions: js,mjs,json
[0] [nodemon] starting `node src/server/index.js`
[0] Listening on port 7777!
[1] clean-webpack-plugin: E:\Users\kwa\Clients\lum\Projects\odinson-project\odinson\ui\dist has been removed.
[1] i ｢wds｣: Project is running at http://localhost:3000/
[1] i ｢wds｣: webpack output is served from /
[1] i ｢wds｣: Content not from webpack is served from E:\Users\kwa\Clients\lum\Projects\odinson-project\odinson\ui
[1] i ｢wdm｣: wait until bundle finished: /
[1] i ｢wdm｣: Hash: 5dbbcad14fe5d31a06ca
[1] Version: webpack 4.43.0
[1] Time: 6358ms
[1] Built at: 07/23/2020 8:47:57 PM
[1]                                 Asset       Size  Chunks             Chunk Names
[1]  0a5c76518a68c185baa2c6744456918c.eot   99.3 KiB          [emitted]
[1]  51ec31f302d0072808e1f83f85fea4cd.ttf   99.1 KiB          [emitted]
[1]                             bundle.js   16.5 MiB    main  [emitted]  main
[1] cef8cdbb9d0ba82e6e19fb0eeba2ac3d.woff   99.2 KiB          [emitted]
[1]                           favicon.ico   1.12 KiB          [emitted]
[1]                            index.html  429 bytes          [emitted]
[1] Entrypoint main = bundle.js
[1] [0] multi (webpack)-dev-server/client?http://localhost:3000 (webpack)/hot/dev-server.js babel-polyfill ./src/client/index.js 64 bytes {main} [built]
[1] [./node_modules/babel-polyfill/lib/index.js] 833 bytes {main} [built]
[1] [./node_modules/babel-polyfill/node_modules/regenerator-runtime/runtime.js] 23.9 KiB {main} [built]
[1] [./node_modules/core-js/fn/regexp/escape.js] 108 bytes {main} [built]
[1] [./node_modules/core-js/shim.js] 8.03 KiB {main} [built]
[1] [./node_modules/react-dom/index.js] 1.33 KiB {main} [built]
[1] [./node_modules/react/index.js] 190 bytes {main} [built]
[1] [./node_modules/webpack-dev-server/client/index.js?http://localhost:3000] (webpack)-dev-server/client?http://localhost:3000 4.29 KiB {main} [built]
[1] [./node_modules/webpack-dev-server/client/overlay.js] (webpack)-dev-server/client/overlay.js 3.51 KiB {main} [built]
[1] [./node_modules/webpack-dev-server/client/socket.js] (webpack)-dev-server/client/socket.js 1.53 KiB {main} [built]
[1] [./node_modules/webpack-dev-server/client/utils/createSocketUrl.js] (webpack)-dev-server/client/utils/createSocketUrl.js 2.91 KiB {main} [built]
[1] [./node_modules/webpack-dev-server/client/utils/log.js] (webpack)-dev-server/client/utils/log.js 964 bytes {main} [built]
[1] [./node_modules/webpack-dev-server/client/utils/reloadApp.js] (webpack)-dev-server/client/utils/reloadApp.js 1.59 KiB {main} [built]
[1] [./node_modules/webpack/hot/dev-server.js] (webpack)/hot/dev-server.js 1.59 KiB {main} [built]
[1] [./src/client/index.js] 181 bytes {main} [built]
[1]     + 714 hidden modules
[1] Child html-webpack-plugin for "index.html":
[1]      1 asset
[1]     Entrypoint undefined = index.html
[1]     [./node_modules/html-webpack-plugin/lib/loader.js!./public/index.html] 552 bytes {0} [built]
[1]     [./node_modules/lodash/lodash.js] 530 KiB {0} [built]
[1]     [./node_modules/webpack/buildin/global.js] (webpack)/buildin/global.js 472 bytes {0} [built]
[1]     [./node_modules/webpack/buildin/module.js] (webpack)/buildin/module.js 497 bytes {0} [built]
[1] i ｢wdm｣: Compiled successfully.
[0] { odinsonQuery: '[tag=/(NN|JJ).*/]* [tag=/N.*/ & lemma=/.*oma|itis/]' }
[0] (node:16060) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'data' of undefined
[0]     at E:\Users\kwa\Clients\lum\Projects\odinson-project\odinson\ui\src\server\index.js:58:29
[0]     at processTicksAndRejections (internal/process/task_queues.js:97:5)
[0] (node:16060) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
[0] (node:16060) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
[0] { odinsonQuery: '[tag=/(NN|JJ).*/]* [tag=/N.*/ & lemma=/.*oma|itis/]' }
[0] (node:16060) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'data' of undefined
[0]     at E:\Users\kwa\Clients\lum\Projects\odinson-project\odinson\ui\src\server\index.js:58:29
[0]     at processTicksAndRejections (internal/process/task_queues.js:97:5)
[0] (node:16060) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 4)
```

**Comment by danebell** [comment]
I'm not at all surprised that the old UI isn't functional anymore. There have been big changes to Odinson since we designed this. I'm not sure how to size this yet, however. We'll have to examine the breaking points, and whether they're limited to the API or what.

**Comment by danebell** [comment]
By the way, if it's at all possible, we should try to use either the simple UI or the IDE, which are under more or less active development. The UI referenced in the documentation isn't active, so it may end up taking a lot of effort for not much payout.

**Comment by myedibleenso** [comment]
I think we can link out to @BeckySharp 's UI in the documentation if it's public and that's what others are using, but my understanding is that licensing issues prevent us from including it in this repo.  A copy of the current UI (the broken one @kwalcock is referencing) has been moved to a separate project (which has no pulse at the moment, but something I'll probably fork in a month or so and fix).  It's in a state of suspended animation right now...  

Basically, we need a PR to remove the source of that UI from lum-ai/odinson and update the docs.
